PORT-2362 Relation delete docs (#138)
port-labs/port-docs: docs/software-catalog/entity/tutorial.md

properties': {"version": "2.29"}
```

### Remove Relations

#### From the API

The API offers several methods to remove the Relations mapping from an an existing Entity:

:::note

> If you set a relation that has a many: true setting to null, it will remove all the relations for that entity.

Reviewer: ```suggestion
If a Blueprint has a Relation of type `"many": true`, by setting the specific Relation key to `null` on the Entity, the mapped related Entities will be removed.
```

:::

##### POST request **(with upsert: true)**

> Make an **HTTP POST** request to the URL `https://api.getport.io/v1/blueprints/{blueprint_identifier}/entities/{entity_identifier}?upsert=true`.

Reviewer: ```suggestion
Make an **HTTP POST** request to the URL `https://api.getport.io/v1/blueprints/{blueprint_identifier}/entities/{entity_identifier}?upsert=true`, for example:
```

> To remove a relation from a `many` relation, for example: `environment`, send a PUT request with the following body:

Reviewer: ```suggestion
To remove a mapped related Entity from a `"many": true` relation, for example: `environment`, send a POST request with the following body:
```

Reviewer: Flip between these 2 examples such that many true is second

```json showLineNumbers
{
  "identifier": "requests-pkg-v2-28",
  "title": "Requests",
  "team": "",
  "properties": {
    ...
  },
  "relations": {
    "environment": [//All the relations identifiers but the one you want to remove]
  }
}
```

> To remove a specific relation, for example: `environment`, send a PUT request with the following body:

Reviewer: ```suggestion
To remove a mapped related Entity from a `"many": false` relation, for example: `environment`, send a POST request with the following body:
```

```json showLineNumbers
{
  "identifier": "requests-pkg-v2-28",
  "title": "Requests",
  "team": "",
  "properties": {
    ...
  },
  "relations": {
    "environment": null
  }
}
```

##### PUT request

> Make an **HTTP PUT** request to the URL `https://api.getport.io/v1/blueprints/{blueprint_identifier}/entities/{entity_identifier}`.

Reviewer: ```suggestion
Make an **HTTP PUT** request to the URL `https://api.getport.io/v1/blueprints/{blueprint_identifier}/entities/{entity_identifier}`, for example:
```

> To remove a relation from a `many` relation, for example: `environment`, send a PUT request with the following body:

Reviewer: Update the text in these following examples to better match the updates to POST upsert

```json showLineNumbers
{
  "identifier": "requests-pkg-v2-28",
  "title": "Requests",
  "team": "",
  "properties": {
    ...
  },
  "relations": {
    "environment": [//All the relations identifiers but the one you want to remove]
  }
}
```

> To remove a specific relation, for example: `environment`, send a PUT request with the following body:

Reviewer: Update the text in these following examples to better match the updates to POST upsert

```json showLineNumbers
{
  "identifier": "requests-pkg-v2-28",
  "title": "Requests",
  "team": "",
  "properties": {
    ...
  },
  "relations": {
    "environment": null
  }
}
```

##### PATCH request

> Make an **HTTP PATCH** request to the URL `https://api.getport.io/v1/blueprints/{blueprint_identifier}/entities/{entity_identifier}`.

Reviewer: ```suggestion
Make an **HTTP PATCH** request to the URL `https://api.getport.io/v1/blueprints/{blueprint_identifier}/entities/{entity_identifier}`, for example:
```

> To remove a relation from a `many` relation, for example: `environment`, send a PATCH request with the following body:

Reviewer: Update the text in these following examples to better match the updates to POST upsert

```json showLineNumbers
{
  "relations": {
    "environment": [//All the relations identifiers but the one you want to remove]
  }
}
```

> To remove a specific relation, for example: `environment`, send a PATCH request with the following body:

Reviewer: Update the text in these following examples to better match the updates to POST upsert

> To remove a mapped related Entity from a `"many": false` relation, for example: `environment`, send a PATCH request with the following body:

Reviewer 2: ```suggestion
To remove a mapped related Entity from a `"many": false` relation (for example: `environment`), send a PATCH request with the following body:
```

- Make an **HTTP DELETE** request to the URL `https://api.getport.io/v1/blueprints/{blueprint_identifier}/entities/{entity_identifier}`

## Remove Relations

### From the API

The API offers several methods to update an existing Entity:

> #### PUT request

Author: Added

Make an **HTTP PUT** request to the URL `https://api.getport.io/v1/blueprints/{blueprint_identifier}/entities/{entity_identifier}` and set the relation value to null.

To remove a specific relation, for example: `environment`, send a PUT request with the following body:

```json showLineNumbers
{
  "identifier": "requests-pkg-v2-28",
  "title": "Requests",
  "team": "",
  "properties": {
    ...
  },
  "relations": {
    "environment": null
  }
}
```

#### PATCH request

Make an **HTTP PATCH** request to the URL `https://api.getport.io/v1/blueprints/{blueprint_identifier}/entities/{entity_identifier}` and set the relation value to null.

To remove a specific relation, for example: `environment`, send a PATCH request with the following body:

```json showLineNumbers
{
  "relations": {

>     "environment": null

Author: Yeah
It will remove all relations
